packages/cfpb-forms/src/atoms/range.less

```diff
@@ -0,0 +1,56 @@
+.rangeTrackStyles() {
+  height: unit((11px / @base-font-size-px), em);
+  width: 100%;
+  background-color: @range-bg-track;
+  border: 1px solid @range-border;
+  border-radius: 8px;
+  cursor: pointer;
+}
+
+.rangeThumbStyles() {
```

Comment:
Let's name this `.u-range-thumb-styles` to align with how we use utility mixins that are used elsewhere in the file (ex. https://github.com/cfpb/design-system/blob/main/packages/cfpb-layout/src/molecules/heroes.less#L73)